File: informe 1/3.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;

int main() {
    srand(time(0));
    int frecuencia[6] = {0};

    for (int i = 0; i < 20000; i++) {
        int cara = rand() % 6; // 0 a 5
        frecuencia[cara]++;
    }

    cout << "Frecuencia de cada cara:\n";
    for (int i = 0; i < 6; i++) {
        cout << "Cara " << (i+1) << ": " << frecuencia[i] << endl;
    }
    return 0;
}

File: informe 1/3.py
import random

frecuencia = [0]*6

for _ in range(20000):
    cara = random.randint(1, 6) # 1 a 6
    frecuencia[cara-1] += 1

print("Frecuencia de cada cara:")
for i, f in enumerate(frecuencia, start=1):
    print(f"Cara {i}: {f}")